prompt-service/src/unstract/prompt_service/services/answer_prompt.py

```diff
@@ -313,3 +314,10 @@ def handle_highlight(
                     if PSKeys.CONFIDENCE_DATA not in metadata:
                         metadata[PSKeys.CONFIDENCE_DATA] = {}
                     metadata[PSKeys.CONFIDENCE_DATA][prompt_key] = confidence_data
+
+    @staticmethod
+    def cut_until_first_bracket(text) -> str:
+        idx_brace = text.find("{")
+        idx_bracket = text.find("[")
+        start = min(i for i in [idx_brace, idx_bracket] if i != -1)
+        return text[start:]
```

Comment:
```suggestion
    def slice_from_first_bracket(text: str) -> str:
        match = re.search(r'[\{\[]', text)
        if match:
            return text[match.start():]
        return text  # or return "" if you'd prefer to return an empty string
```
@Deepak-Kesavan the implementation is to extract text from the first bracket right? If so, I think the function is named incorrectly.
Also it'll fail with a value error if the text does not have any of those braces. Consider the regex implementation instead

Reply:
@Deepak-Kesavan the name of the function is misleading btw. Please change that too

Reply:
@chandrasekharan-zipstack 

Using re.search for this case is unnecessary and introduces overhead. We're only checking for the first occurrence of one of two fixed characters ({ or [), which is more efficiently and clearly handled with str.find. Regex adds complexity and cost without benefit here.

I have fixed the case where brackets are not found in current approach itself.